backend/events/apps.py

```diff
@@ -49,6 +49,10 @@ def on_tuples(cc_event, block_number, tx_id, tx_status):
     worker_queue = f"{LEDGER['name']}.worker"
 
     for tuple_type, _tuples in payload.items():
+
+        if tuple_type not in ['traintuple', 'compositeTraintuple', 'aggregatetuple']:
+            continue
```

Comment:
What's the reason for adding these 2 lines? It looks like testtuple won't be executed anymore.

EDIT:
I get it, it's because we want to filter `computePlan` (it has been added in the companion PR). In this case I would add a comment to explain the goal of this filter. Or maybe do the opposite filter: `if tuple_type == 'computePlan'` ?

Reply:
I'm going to convert this code to a switch/case, it'll make things much clearer!